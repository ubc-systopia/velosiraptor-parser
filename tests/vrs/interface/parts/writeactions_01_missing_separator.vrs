WriteActions { A -> B  A -> B; }