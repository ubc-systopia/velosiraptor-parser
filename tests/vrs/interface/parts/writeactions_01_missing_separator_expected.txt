error: expected `;`, found `A`
     --> tests/vrs/interface/parts/writeactions_01_missing_separator.vrs:1:24
      |
    1 |  WriteActions { A -> B  A -> B; }
      |                         ^ help: add `;` here.
